### ISSUE TITLE
Wire1 not compatible with TeensyLC

### ISSUE THREAD

**Issue by nseidle** [RoboticMaterials/FA-I-sensor#3]
Hi - I got my kit a few months ago and am returning to testing. The example code doesn't work. Problem is [here](https://github.com/RoboticMaterials/FA-I-sensor/blob/master/force_proximity_eval/force_proximity_eval.ino#L19).

Changing from Wire1 to Wire fixes the problem. Perhaps you've written the code to work with a Teensy 3.x that has multiple I2C ports? Not sure.

**Comment by correll** [comment]
Yes, the teensy lc has two i2c ports and I use the second one by default as those pins are next to the power ones. This is how I ship the devices.

What's your status? One of my students saw it at a lab tour.
